### ISSUE TITLE
Accessibility: Left navigation does not have a landmark/header

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#151]
Currently, the left menu does not have a heading, which makes it difficult do identify with a screen reader.
